isilon_create_users.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 ###########################################################################
-##  Script to create Hadoop users on Isilon.
```

Comment:
It looks like your editor replaced regular spaces with UTF-8 HTML non-breaking spaces:
``` python
>>> r'   '
'   '
>>> r'   '
'\xc2\xa0\xc2\xa0 '
```
Those are obscuring the actual changes you made to the script.
Try running these commands on your clone:
``` sh
sed -i 's/\xC2\xA0/ /g' isilon_create_users.sh
git commit -a --fixup 70d9a13
git push
```